test/screens/play/play_page_test.dart

```diff
@@ -34,4 +40,21 @@ void main() {
     expect(find.byType(ScryingPage), findsNothing);
     expect(find.byType(PlayerHand), findsOneWidget);
   });
+
+  testWidgets('should change colour based on the card type displayed',
+      (WidgetTester tester) async {
+    var cardFinder = find.byType(Card);
```

Comment:
This can be `final` now that we're not reassigning it. Also think formatting/indenting has gone a bit wonky, but LGTM!